tests/irq_cpp/Makefile

```
@@ -0,0 +1,12 @@
+include ../Makefile.tests_common
+FEATURES_REQUIRED += cpp
```

```suggestion
FEATURES_REQUIRED += cpp
FEATURES_REQUIRED += libstdcpp
```

`ccpunit.hpp` includes `<array>`, which is part of the standard C++ lib (and therefore not available on AVR).